cerebrium/container-images/defining-container-images.mdx

```diff
@@ -0,0 +1,288 @@
+---
+title: Defining Container Images
+---
+
+## Introduction
+
+Cerebrium simplifies the deployment and running of machine learning apps by abstracting infrastructure management into configuration, allowing engineering teams to focus on what matters most - delivering value to customers using application code. A single TOML file manages environment setup, deployments, and scaling - tasks that typically require dedicated teams.
+
+## Why TOML?
+
+While Python decorators offer programmatic configuration, they scatter infrastructure settings throughout code files, making changes risky and review difficult. TOML centralizes all configuration in one place, making it easier to track changes and maintain consistency. Its straightforward syntax prevents the accidental complexity that often comes with code-based configuration, while its hierarchical structure naturally maps to modern app requirements.
+
+## Understanding containers on Cerebrium
```

Comment:
seems strange to have a introduction and understanding containers. Why not just put it all under introduction